app/src/main-process/squirrel-updater.ts

```diff
@@ -18,6 +18,9 @@ const exeName = Path.basename(process.execPath)
  */
 export function handleSquirrelEvent(eventName: string): Promise<void> | null {
   switch (eventName) {
+    case '--squirrel-firstrun':
+      return Promise.resolve()
```

Comment:
Doesn't this mean the app will quit once the promise resolves? https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/using/custom-squirrel-events-non-cs.md#application-startup-commands makes it sound like we shouldn't.